PostFreeSurfer/PostFreeSurferPipelineLongPrep.sh

```diff
@@ -406,19 +406,24 @@ if (( TemplateProcessing ==  1 )); then
         Timepoint_brain_mask_MNI="${AtlasSpaceFolder_timepoint}"/"$T1wImageBrainMask".nii.gz
 
         #link altas transforms to the timepoint directory.
-        if [ -d "$AtlasSpaceFolder_timepoint/xfms" ]; then 
+        if [ -d "$AtlasSpaceFolder_timepoint/xfms" ]; then
             rm -rf "$AtlasSpaceFolder_timepoint/xfms"
-        fi                
-        ln -sf "${AtlasSpaceFolder_template}/xfms" "${AtlasSpaceFolder_timepoint}/xfms"
+        fi
+        mkdir -p ${AtlasSpaceFolder_timepoint}
+        ln -sf "${AtlasSpaceFolder_template}/xfms" "${AtlasSpaceFolder_timepoint}/"
 
         #one mask for all timepoints.
         cp ${AtlasSpaceFolder_template}/"$T1wImageBrainMask".nii.gz "$Timepoint_brain_mask_MNI"
         cp "${T1w_dir_template}/$T1wImageBrainMask".nii.gz "$Timepoint_brain_mask_acpc_dc"
-        
+
         #mask the acpc_dc space images.
         ${FSLDIR}/bin/fslmaths $T1w_dir_long/T1w_acpc_dc -mas "$Timepoint_brain_mask_acpc_dc" $T1w_dir_long/T1w_acpc_dc_brain
         ${FSLDIR}/bin/fslmaths $T1w_dir_long/T1w_acpc_dc_restore -mas "$Timepoint_brain_mask_acpc_dc" $T1w_dir_long/T1w_acpc_dc_restore_brain
 
+        # Bias field in MNI space.
+        ${FSLDIR}/bin/applywarp --rel --interp=spline -i $T1w_dir_long/BiasField_acpc_dc -r ${T1wTemplate} -w ${AtlasSpaceFolder_template}/$WARP \
+            -o ${AtlasSpaceFolder_timepoint}/BiasField
+
```

Comment:
This is also already in master, it should not have been repeated in a PR commit (don't fix problems that are already in master in an unrelated PR, keep PRs single-purpose).  If the only other edit (commenting out a stage) is also wrong, I suggest reverting this file to the version in master, this should do the trick (while on the PR branch):

`git checkout master -- PostFreeSurfer/PostFreeSurferPipelineLongPrep.sh`

Then stage (or it might already be staged) and commit.

Reply:
Err, scratch that, having a copy of the master version when the branch point is older is the problem here...

Reply:
I think I'll just remove the last commit and rebase.

Reply:
Please save any commits on this branch you haven't pushed (`git branch save-commits`), uncommitted work (`git stash`) and reset your diffusion branch to origin:

```
git checkout --detach
git remote update
git branch -f diffusion_preprocessing_longitudinal origin/diffusion_preprocessing_longitudinal
git checkout diffusion_preprocessing_longitudinal
```